src/config/mockMode.js

```
@@ -64,27 +71,198 @@ class MockModeManager {
+            }
+            
+            const chainableInsert = {
+              select: (columns = '*') => ({
+                single: () => {
+                  const result = Array.isArray(data) ? { ...data[0], id: 1 } : { ...data, id: 1 };
+                  return Promise.resolve({
+                    data: result,
+                    error: null
+                  });
+                },
+                then: (resolve) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)